### ISSUE TITLE
mic event doesn't work in Blockly

### ISSUE THREAD

**Issue by motib** [Mobsya/blockly#12]
The mic event doesn't work because the threshold is initialized to 0 and can't be changed.

**Comment by stephanemagnenat** [comment]
Not exactly. The microphone threshold can be set using the "set mipcrophone threshold to" block. But I agree that it should be initialized to a sane value.
